stress/shell/src/main/java/alluxio/stress/cli/StressJobServiceBench.java

```diff
@@ -245,53 +245,38 @@ private void runInternal() throws Exception {
             mContext.getStartMs(), CommonUtils.getCurrentMs()));
       }
       CommonUtils.sleepMs(waitMs);
-      applyOperation(mPath);
+      long recordMs = mContext.getStartMs() + FormatUtils.parseTimeSize(mParameters.mWarmup);
+      mResult.setRecordStartMs(recordMs);
+      while (true) {
```

Comment:
I think the problem of putting while loop here is we only createFile and run distributedLoad once although we are sending request constantly for NoOp.

Reply:
My purpose of the loop here is to keep executing when warm up. 
I am also a little confused. It seems that each thread should run multiple times, relying on `mContext.getRateLimiter().acquire()` to limit the running time(`--target-throughput`)

Reply:
thanks,I will take a look

Reply:
@jja725 I have reverted this part of the change. I added a record of createFiles start time and number of successes.